.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,494 @@
+update_displayed_attributes_1: |-
+  let displayed_attributes = &[
+    "title",
+    "description",
+    "release_date",
+    "rank",
+    "poster"
+  ];
+
+  let progress: Progress = movie_index.set_displayed_attributes(displayed_attributes).await.unwrap();
+reset_displayed_attributes_1: |-
+  let progress: Progress = movie_index.reset_displayed_attributes().await.unwrap();
+get_accept_new_fields_1: |-
+  let accept_new_fields: bool = movie_index.get_accept_new_fields().await.unwrap();
+update_accept_new_fields_1: |-
+  let progress: Progress = movie_index.set_accept_new_fields(false).await.unwrap();
+get_index_stats_1: |-
+  let stats: IndexStats = movie_index.get_stats().await.unwrap();
+get_indexes_stats_1: |-
+  let stats: ClientStats = client.get_stats().await.unwrap();
+get_health_1: |-
+  // get_health() return an Err() if the server is not healthy, so this example would panic due to the unwrap
+  client.get_health().await.unwrap();
+update_health_1: |-
+  client.set_health(false).await.unwrap();
+get_version_1: |-
+  let version: Version = client.get_version().await.unwrap();
+get_pretty_sys_info_1: |-
+  // This route is not supported because it will be soon removed
+get_sys_info_1: |-
+  // This route is not supported because it will be soon removed
+distinct_attribute_guide_1: |-
+  let jacket_index: Index = client.get_index("jackets").await.unwrap();
+  let progress: Progress = jacket_index.set_distinct_attribute("product_id").await.unwrap();
+field_properties_guide_searchable_1: |-
+  let searchable_attributes = &[
+    "uid",
+    "movie_id",
+    "title",
+    "description",
+    "poster",
+    "release_date",
+    "rank"
+  ];
+
+  let progress: Progress = movie_index.set_searchable_attributes(searchable_attributes).await.unwrap();
+field_properties_guide_displayed_1: |-
+  let displayed_attributes = &[
+    "title",
+    "description",
+    "poster",
+    "release_date",
+    "rank"
+  ];
+
+  let progress: Progress = movie_index.set_displayed_attributes(displayed_attributes).await.unwrap();
+filtering_guide_1: |-
+  // Build the query
+  let query: Query = Query::new("Avengers")
+    .with_filters("release_date > 795484800");
+  
+  // Apply the query and get results
+  let results: SearchResults<Movie> = movie_index.search(&query).await.unwrap();
+filtering_guide_2: |-
+  // Build the query
+  // Warning: the testing database you downloaded does not contains the fields used in the filters so this example will not work on this database
+  let query: Query = Query::new("Batman")
+    .with_filters("release_date > 795484800 AND (director = \"Tim Burton\" OR director = \"Christopher Nolan\")");
+  
+  // Apply the query and get results
+  let results: SearchResults<Movie> = movie_index.search(&query).await.unwrap();
+filtering_guide_3: |-
+  // Build the query
+  // Warning: the testing database you downloaded does not contains the field "director" used in the filters so this example will not work on this database
```

Comment:
Same @bidoubiwa? What do you think about this comment?

Reply:
I aggree with you, not needed.